pkg/build/buildutil/util.go

```diff
@@ -42,8 +43,22 @@ const (
 	sysConfigConfigMapSuffix = "sys-config"
 	// GlobalCAConfigMapKey is the key into the config map data for the injected CA data
 	GlobalCAConfigMapKey = "ca-bundle.crt"
+
+	// this is the regexp for the k8s pod label validation
+	// unfortunately a public variable is not exposed for anything related to the regexp,
```

Comment:
perhaps an opportunity for an upstream contribution, but IMO we don't gate our fix based on that, as per the bugzilla this came from a CI failure to build

Reply:
I found the public API in k8s that allows for us to validate

Will update PR shortly.